src/DependencyInjection/CmfSonataAdminIntegrationExtension.php

```diff
@@ -16,19 +16,46 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Cmf\Bundle\SonataAdminIntegrationBundle\DependencyInjection\Factory\AdminFactoryInterface;
+use Symfony\Cmf\Bundle\SeoBundle\CmfSeoBundle;
 
 /**
  * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ * @author Wouter de Jong <wouter@wouterj.nl>
  */
 class CmfSonataAdminIntegrationExtension extends Extension
 {
+    /**
+     * @var AdminFactoryInterface[]
+     */
+    private $factories;
+
+    /**
+     * @param null|AdminFactoryInterface[] $factories A list of Admin factories
```

Comment:
No, factories are just Names mapped to admin factories (e.g. `'seo' => new Factory\SeoAdminFactory()`).

Your confusing `$bundles` with `$this->factories`.